Update isce2_topsapp.__main__.gunw_slc to accept either `--reference-scenes` or `--reference-date` (#207)
ACCESS-Cloud-Based-InSAR/DockerizedTopsApp: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.6.0]

### Added
* `isce2_topsapp.__main__.gunw_slc` now accepts `--reference-date` and `--secondary-date` parameters as alternatives to
  `--reference-scenes` and `--secondary-scenes`. When provided, `gunw_slc` will process all Sentinel-1 IW SLC scenes for

>   the given date and `--frame-id` in order to generate a standard ARIA_S1_GUNW product.

Reviewer: Maybe we should avoid implying that using the date inputs is sufficient to achieve a standard product? Since there are other criteria (e.g. `--min-frame-coverage`) that must also be met.
```suggestion
  the given date and `--frame-id` in order to generate an ARIA_S1_GUNW product.
```
